draft-bclp-green-terminology.md

```diff
@@ -173,17 +173,27 @@ The proposed metric for router and Ethernet switches is
 
 Where Ti is weighted throughput, Pw is weighted power (energy consumption rate)
 
+                        Ti = a*Tu1 + b*Tu2 + c*Tu3
+                        Pw = a*Pu1 + b*Pu2 + c*Pu3
+
+where (a, b, c) is the relative weight at different usage percentage, a+b+c=1
+
+(Pu1, Pu2, Pu3) is the power at different usage percentage
+
+(Tu1, Tu2, Tu3) is the throughput at different usage percentage
+
 Metric for small network devices:
 
 The metrics adopted for small networking devices intended for home/domestic or small office use is
```

Comment:
"adopted": consider adding a reference or clarify this was adopted by whom.

Reply:
these definitions are from {{L.1310}}, added references.

Reply:
works for me.